Misc/NEWS.d/next/Core and Builtins/2021-01-04-23-54-34.bpo-42819.4KO6wU.rst

```diff
@@ -0,0 +1,3 @@
+Prevents bracketed paste from being enabled in the interactive interpreter,
+even if it's set in the inputrc or is enabled by default (eg GNU Readline
+8.1). Patch by Dustin Rodrigues.
```

Comment:
Hum, I suggest to say that Python now explicitly disables the bracketed mode when the readline module is imported, but also in the readline.read_init_file() function.